.editorconfig

```diff
@@ -0,0 +1,53 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+[*.nix]
+indent_size = 2
+
+# Ignore paths
+[lua/nio/**]
+charset = unset
+end_of_line = unset
+insert_final_newline = unset
```

Comment:
I am not familiar with the `unset`. What's the advantage of defining those ?

Reply:
editorconfig doesn't have an "ignore" config.
This is a workaround for that.